cypress/integration/Crosscutting/HttpStatusCodeTest/index.js

```diff
@@ -0,0 +1,11 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Then('correct status code {string} is received for the {string}', (code, path) => {
```

Comment:
dont forget to change codes' type from {string} to {int}

Reply:
done